ISSUE_TEMPLATE.md

```diff
@@ -10,91 +10,13 @@ Make sure you have provided the following information:
-###### CVE-2020-25632, CVE-2020-25647, CVE-2020-27749, CVE-2020-27779,
-###### CVE-2021-20225, CVE-2021-20233, CVE-2020-10713, CVE-2020-14308,
-###### CVE-2020-14309, CVE-2020-14310, CVE-2020-14311, CVE-2020-15705,
-###### ( July 2020 grub2 CVE list + March 2021 grub2 CVE list )
-###### and if you are shipping the shim_lock module CVE-2021-3418
-###### fixed ?
-`[your text here]`
-
-###### "Please specifically confirm that you add a vendor specific SBAT entry for SBAT header in each binary that supports SBAT metadata
-###### ( grub2, fwupd, fwupdate, shim + all child shim binaries )" to shim review doc ?
-###### Please provide exact SBAT entries for all SBAT binaries you are booting or planning to boot directly through shim
-`[your text here]`
-
-##### Were your old SHIM hashes provided to Microsoft ?
-`[your text here]`
-
-##### Did you change your certificate strategy, so that affected by CVE-2020-14372, CVE-2020-25632, CVE-2020-25647, CVE-2020-27749,
-##### CVE-2020-27779, CVE-2021-20225, CVE-2021-20233, CVE-2020-10713,
-##### CVE-2020-14308, CVE-2020-14309, CVE-2020-14310, CVE-2020-14311, CVE-2020-15705 ( July 2020 grub2 CVE list + March 2021 grub2 CVE list )
-##### grub2 bootloaders can not be verified ?
-`[your text here]`
-
-##### What exact implementation of Secureboot in grub2 ( if this is your bootloader ) you have ?
-##### * Upstream grub2 shim_lock verifier or * Downstream RHEL/Fedora/Debian/Canonical like implementation ?
-`[your text here]`
-
-###### What is the origin and full version number of your bootloader (GRUB or other)?
-`[your text here]`
-
-###### If your SHIM launches any other components, please provide further details on what is launched
-`[your text here]`
-
-###### If your GRUB2 launches any other binaries that are not Linux kernel in SecureBoot mode,
-###### please provide further details on what is launched and how it enforces Secureboot lockdown
-`[your text here]`
-
-###### If you are re-using a previously used (CA) certificate, you
-###### will need to add the hashes of the previous GRUB2 binaries
-###### exposed to the CVEs to vendor_dbx in shim in order to prevent
-###### GRUB2 from being able to chainload those older GRUB2 binaries. If
-###### you are changing to a new (CA) certificate, this does not
-###### apply. Please describe your strategy.
-`[your text here]`
-
-###### How do the launched components prevent execution of unauthenticated code?
+-------------------------------------------------------------------------------
+### What organization or people are asking to have this signed?
+-------------------------------------------------------------------------------
 `[your text here]`
 
-###### Does your SHIM load any loaders that support loading unsigned kernels (e.g. GRUB)?
+-------------------------------------------------------------------------------
+### What product or service is this for?
+-------------------------------------------------------------------------------
 `[your text here]`
 
```

Comment:
On [line 3](https://github.com/rhboot/shim-review/pull/207/files#diff-e5aa817955e06f89b32731f49a1ccf2bcd70eb907e023ca0766899cf64d7f9aaR3) it says to include a link to your fork of shim-review. Should we have a specific question here so it's clear where to put it/what format?